packages/keystone/apolloErrorFormatter.spec.js

```diff
@@ -287,13 +309,740 @@ describe('safeFormatError hide=false', () => {
                 'data': { bar: '33' },
                 'time_thrown': expect.stringMatching(/^\d\d\d\d-\d\d-\d\dT\d\d:\d\d:\d\d/),
             },
-            'extensions': {},
         })
     })
     test('safeFormatError(null)', () => {
-        expect(safeFormatError(null)).toMatchObject({
+        expect(safeFormatError(null)).toEqual({
             'message': 'null',
             'name': 'NonError',
+            'stack': expect.stringMatching(/^NonError: null/),
+        })
+    })
+    test('safeFormatError(KeystoneAccessDeniedError)', () => {
+        const data = { type: 'query', target: 'user' }
+        const internalData = { ...GQL_KEYSTONE_INTERNAL_DATA_EXAMPLE }
+        const original = new AccessDeniedError({ data, internalData })
+        const error = new GraphQLError('GraphQLError1', [GQL_FIELD_NODE_EXAMPLE], GQL_SOURCE_EXAMPLE, null, ['field'], original, {})
+        expect(safeFormatError(error)).toEqual({
+            'locations': [{ column: 5, line: 3 }],
+            'message': 'GraphQLError1',
```

Comment:
Why message has nothing about access denied? Was it intentional?

Reply:
In real world case the error message will pass to `GraphQLError` from parent error. And you will see the `You do not have access to this resource`. I've add some notes about this case